zipline/testing/fixtures.py

```diff
@@ -614,12 +614,7 @@ def _make_equity_daily_bar_from_minute(cls):
             'high': 'max',
             'low': 'min',
             'close': 'last',
-            # TODO: Change test data so that large minute volumes are not used,
-            # so that 'sum' can be used without going over the uint limit.
-            # When that data is changed, this function can and should be moved
-            # to the `data` module so that loaders and tests can use the same
-            # source from minute logic.
-            'volume': 'last'
```

Comment:
Thanks for spotting that.
Yes, I will take our internal resampling and put in a module in data, then move our internal code to use that new function.